src/main/java/core/basesyntax/serviceinterfaces/CsvMapper.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.serviceinterfaces;
+
+import java.util.List;
+
+public interface CsvMapper {
```

Comment:
I guess it is better to name it just Mapper because `Csv` prefix is more about specification and implementation